src/main/java/io/confluent/idesidecar/restapi/filters/AccessTokenFilter.java

```diff
@@ -81,7 +81,11 @@ public void enforceAccessToken(RoutingContext rc) {
 
         String expectedToken = accessTokenBean.getToken();
         if (expectedToken == null) {
-          Log.error("Request for " + requestPath + " made before handshake route hit! Rejecting.");
+          if (enabled.get()) {
+            Log.error("Request for " + requestPath + " made before handshake route hit! Rejecting.");
+          } else {
+            Log.debug("Request for " + requestPath + " made without handshake.");
+          }
```

Comment:
This is another log message that shows up for every API call made in our integration tests (where `@TestProfile(NoAccessFilterProfile.class)` is used), and inaccurately says the request is rejected. Again, the production log messages are very useful, but are not nearly as important during integration tests.